journal/src/main/java/io/camunda/zeebe/journal/file/SegmentedJournal.java

```diff
@@ -438,6 +438,7 @@ JournalSegment createSegment(final JournalSegmentDescriptor descriptor) {
       throw new JournalException(e);
     } finally {
       try {
+        channel.force(true);
```

Comment:
Let's drop this and create a separate PR to deal with the log stuff. I'd like to take that opportunity to refactor the segment creation anyway.

Reply:
Do you want to create an issue for that?

Reply:
I can do that :+1: